src/c20_client/client.py

```diff
@@ -25,14 +26,29 @@ def do_job(api_key):
         print("Getting job...\n")
         job = requests.get('http://capstone.cs.moravian.edu/get_job')
         job = job.json()
-        print("Job aquired: " + json.dumps(job) + "\n")
+        print("Job acquired: " + json.dumps(job) + "\n")
 
     except Exception:
         raise NoConnectionError
 
     get_result_for_job(job, api_key)
 
 
+def do_multiple_job(api_key):
+    """
+       Gets multiple job from the server
+       and handles the job based on the type of job
+    """
+    try:
+        print("Getting job...\n")
+        while True:
+            do_job(api_key)
+            time.sleep(3.66)
```

Comment:
3.66 is a "magic number".   At the very least it should be made into a variable with a descriptive name.  It would be better to factor this out into a separate class - this would allow us to create a mock object and (possibly) allow us to test this code.

Reply:
3.66 is set as a descriptive variable as WAITING_TIME_FOR_EACH_CALL with a note on how it is computed.
The functions are placed in a different file as well.